plugins/auth-backend-module-atlassian-provider/src/authenticator.ts

```diff
@@ -49,13 +72,14 @@ export const atlassianAuthenticator = createOAuthAuthenticator({
           baseURL: baseUrl,
           authorizationURL: `${baseUrl}/authorize`,
           tokenURL: `${baseUrl}/oauth/token`,
-          profileURL: `${baseUrl}/api/v4/user`,
+          profileURL: 'https://api.atlassian.com/me',
+          scope: config.getOptionalString('additionalScopes')?.split(' ') || [],
```

Comment:
Yes, it does pass.

Reply:
@Rugvip : I probably misunderstood your initial question. The scopes that we want in the token is the combination of:
1) the minimum set of scopes defined in the authenticator (we now have 4, since I had to add `read:me` for the profile lookup to work and
2) the scopes defined in the configuration (in my use case, I ask for additional Jira permissions)

Btw, this is checked in the 2 test cases. Makes sense?

Reply:
Oh I see, it seems like the strategy might require the scope option. That's a lil bit silly, but if we have to pass something here it could for example be `scope: 'dummy-ignored'`.

If a scope option is provided to the `authenticate` method then that scope will be used instead of the configured one, there's no merging. See https://github.com/jaredhanson/passport-oauth2/blob/be9bf58cee75938c645a9609f0cc87c4c724e7c8/lib/strategy.js#L231

We'll always be providing a `scope` option to `authenticate`, which means that this option is always ignored.

Reply:
Regarding the `additionalScopes` config, that's already picked up and passed along by the `@backstage/plugin-auth-node` library as input to the methods of the authenticator.